packages/terra-form-input/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Removed
```

Comment:
```suggestion
* Changed
```

This is a change to conditionally apply the class name. It doesn't remove functionality. Removed is slightly misleading.